src/QsCompiler/Core/SyntaxTreeTransformation.fs

```diff
@@ -4,14 +4,16 @@
 namespace Microsoft.Quantum.QsCompiler.Transformations.Core
 
 
+// setup for syntax tree transformations with internal state
+
 type QsSyntaxTreeTransformation<'T> private (state : 'T, unsafe : string) =
```

Comment:
Can this be renamed `SyntaxTreeTransformation` since none of the other classes have the Qs prefix?

Reply:
Yes, I forgot to rename it. Thanks!